notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>"For serving, we need to set up the infrastructure, but by using Vertex AI service, which autoscale depending on demands, we can significantly reduce infrastructure setup and maintenance efforts. " &gt;&gt; "Serving machine learning models requires infrastructure. Vertex AI makes this simple by providing autoscaling services that reduce setup and maintenance effort." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Sounds clearer! Thanks.</p>